scripts/dockerfiles/readiness-probe.sh

```diff
@@ -40,14 +41,17 @@ if test -n "${FISSILE_ACTIVE_PASSIVE_PROBE:-}" ; then
     trap update_readiness EXIT
 fi
 
+if [ ! -r /etc/monitrc ] ; then
+    echo "Waiting for monit to start"
+    exit 1
+fi
+
 # Grab monit port
 monit_port=$(awk '/httpd port/ { print $4 }' /etc/monitrc)
 
 # Check that monit thinks everything is ready
 curl -s -u admin:"${MONIT_PASSWORD}" http://127.0.0.1:"${monit_port}"/_status | gawk '
-    BEGIN                                                     { status = 0 }
-    $1 == "status" && $2 != "running" && $2 != "accessible"   { print ; status = 1 }
-    END                                                       { exit status }
+    $1 == "status" && $2 != "running" && $2 != "accessible"   { print "Waiting for monit to be ready"; exit 1 }
```

Comment:
Use the same message regardless of which job is failing (so things can be collapsed)

Reply:
Isn't it weird to exit from `gawk`?

Reply:
Not really; it's exiting `gawk` early from `gawk`. It just happens (separately) to also exit the surrounding shell when `gawk` returns non-zero.

Reply:
I see!